dependencyManagement/build.gradle.kts

```
@@ -17,6 +17,9 @@ rootProject.extra["otelVersion"] = otelVersion
 // Need both BOM and -all
 val groovyVersion = "2.5.11"
 
+rootProject.extra["caffeine2Version"] = "2.9.2"
+rootProject.extra["caffeine3Version"] = "3.0.3"
```

3.0.4 was released just a few days ago 😄 
```suggestion
rootProject.extra["caffeine3Version"] = "3.0.4"
```